README.md

```diff
@@ -2,157 +2,136 @@
+1.  Create a [Node.js Web App] in Azure.
 
-1. This repo is ready to be deployed using local git. Read this tutorial to get more information on [how to push using local git through portal](https://docs.microsoft.com/en-us/azure/app-service/app-service-deploy-local-git)
+2.  Set environment variables in the **Settings** > **Configuration** > **Application Settings** of your Web App. You can also change the value of the variables from `null` in the index.js file.
 
-## At a glance
-
-Using the `loginWithAppServiceMSI()` method from [ms-rest-azure](https://www.npmjs.com/package/ms-rest-azure) will autodetect if you're on a WebApp and get the token from the MSI endpoint. Then, the code is simply:
-
-```javascript  
-    function getKeyVaultCredentials(){
-        return msRestAzure.loginWithAppServiceMSI({resource: 'https://vault.azure.net'});
-    }
-
-    function getKeyVaultSecret(credentials) {
-        let keyVaultClient = new KeyVault.KeyVaultClient(credentials);
-        return keyVaultClient.getSecret(KEY_VAULT_URI, 'secret', "");
-    }
-
-    getKeyVaultCredentials().then(
-        getKeyVaultSecret
-    ).then(function (secret){
-        console.log(`Your secret value is: ${secret.value}.`);
-    }).catch(function (err) {
-        throw (err);
-    });
-```
-
-If you want to execute this same code in your local environment machine, just use the appropriate login method.
-
-If you need a fallback mechanism to allow this code to switch automatically from MSI to another approach, you can test for environment variables:
-
-```javascript
-function getKeyVaultCredentials(){
-  if (process.env.APPSETTING_WEBSITE_SITE_NAME){
-    return msRestAzure.loginWithAppServiceMSI({resource: 'https://vault.azure.net'});
-  } else {
-    return msRestAzure.loginWithServicePrincipalSecret(clientId, secret, domain);
-  }
-}
-```
+3.  This repository is ready to be deployed using local git. Read this tutorial to get more information on [how to push using local git through portal].
 
 ## Summary
-
-The web app was successfully able to get a secret at runtime from Azure Key Vault using your developer account during development, and using MSI when deployed to Azure, without any code change between local development environment and Azure. 
-As a result, you did not have to explicitly handle a service principal credential to authenticate to Azure AD to get a token to call Key Vault. You do not have to worry about renewing the service principal credential either, since MSI takes care of that.
-
-## Azure Functions
-
-Azure Functions being powered by Azure WebApp, MSI is also available. You can copy the relevant code from the example into your Azure Functions with the right import.
+The Web App was successfully able to get a secret at runtime from Azure Key Vault using your developer account during development, and using Azure Managed Identities when deployed to Azure, without any code change between local development environment and Azure. 
```

Comment:
For the comment above, how about we take this summary and move it to the top and use future tense.